_posts/2018-06-13-Serverless-Basic-Authentication-Custom-Authorizer.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: Serverless Basic Authentication using a Custom Authorizer
+banner: /assets/posts/2018-06-13-Serverless-Basic-Authentication-Custom-Authorizer/banner.png
+github: svdgraaf/serverless-basic-authentication
+author: svdgraaf
+---
+
+In a recent project, we needed our api's to be able to work with external systems. These systems only supported http basic authentication (eg: username/password) for integrating with external systems.
+
+Of course, Basic HTTP Authentication is the easiest and most straight forward way to integrate things and comes build in, in almost any http client.
```

Comment:
"built in",

Suggestion for readability: 
"Of course, Basic HTTP Authentication is the easiest and most straight forward way to integrate things, and it's built into almost any http client.